### PR TITLE
Adding Multiple events

### PR DESCRIPTION
**Pull request by Emersont1**
Adding all the events via this pull request

### CODE REVIEW

**Review by tashBayliss — DISMISSED** [review]

File: src/_events/2020-ff-quirks/summary.md [hunk 1]
```diff
@@ -0,0 +1 @@
+A look at the software quirks of the Final Fantasy Games
```
Comment (erinosher) [inline src/_events/2020-ff-quirks/summary.md]: Can remove for now, please

File: src/_events/2020-ff-quirks/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+- id: 2020-ff-quirks
+  name: Final Fantasy Quirks
+  start: '2020-10-11 19:00:00'
+  end: '2020-10-11 20:30:00'
+  location: An internet connection near you
+  mapLink: https://www.google.co.uk/maps/place/52%C2%B057'12.6%22N+1%C2%B011'14.1%22W/@52.953488,-1.1877845,17.15z/data=!3m1!4b1!4m6!3m5!1s0x0:0x0!7e2!8m2!3d52.9534883!4d-1.1872358
+  summary: 2020-ff-quirks/summary.md
+  description: 2020-ff-quirks/description.md
+  banner: 2020-ff-quirks/banner.jpg
```
Comment (erinosher) [inline src/_events/2020-ff-quirks/manifest.yml]: Can remove for now, please

File: src/_events/2020-ff-quirks/description.md [hunk 3]
```diff
@@ -0,0 +1 @@
+ From speedrunners to lore nerds and all sorts in between push and break games, taking them to pieces to see how they work. Join Alexi and Leanne on a journey through the bugs and exploits that the Final Fantasy community have found and dive into the code and design to see how they work and what lead to these game breaking exploits shipping.
```
Comment (erinosher) [inline src/_events/2020-ff-quirks/description.md]: Can remove for now, please

**Review by erinosher — CHANGES_REQUESTED** [review]
Other than the FF bit needing to be removed, @AksanDotDev hasn't confirmed that date yet, it looks good to me

File: src/_events/2020-ff-quirks/manifest.yml (hunk 2 above)
Comment (erinosher) [reply src/_events/2020-ff-quirks/manifest.yml]: Also, the date is off by a month

**Review by erinosher — APPROVED** [review]

**Pull request merged**